src/uiSelectDirective.js

```diff
@@ -83,8 +83,8 @@ uis.directive('uiSelect',
             $select.sortable = sortable !== undefined ? sortable : uiSelectConfig.sortable;
         });
 
-        scope.$watch(function () { return scope.$eval(attrs.disableBackspaceReset); }, function(newVal) {
-          $select.disableBackspaceReset = newVal !== undefined ? newVal : uiSelectConfig.disableBackspaceReset;
+        scope.$watch(function () { return scope.$eval(attrs.backspaceReset); }, function(newVal) {
```

Comment:
Can you use `attrs.$observe(...)` here instead? Only `$eval` when value is different. (See `resetSearchInput` a few lines down)